Makefile

```diff
@@ -97,19 +97,10 @@ lint: ## Run linter
 	golangci-lint run
 
 buf-gen: ## Run buf lint format generate
-	rm -rf api/{google,grafeas,validate} || true
-	rm -rf internal/api/{google,grafeas,validate} || true
-	buf mod update api
-	buf export buf.build/envoyproxy/protoc-gen-validate -o api
```

Comment:
Don't need `buf export` anymore. The Buf editor plugins will index dependencies for us.

https://plugins.jetbrains.com/plugin/19147-buf-for-protocol-buffers
https://marketplace.visualstudio.com/items?itemName=bufbuild.vscode-buf